releasenotes/notes/fix-circuit-drawer-for-qc-params-e78c67310ae43ccf.yaml

```diff
@@ -0,0 +1,4 @@
+fixes:
+  - |
+    Fixes the circuit drawer from displaying parameters when they are of type `QuantumCircuit` 
+    since this resulted in an illegible drawing.
```

Comment:
Just a small thing. Release notes should have a blank line at the end. The lint catcher didn't get it, but it might in a docs build.

Reply:
Just to clarify - it's just a trailing newline that's missing, though if your editor doesn't add them automatically I guess it probably appears as a blank line.  I don't _think_ it will cause any failures in the build, but I could be wrong.